lib/twilio/eventpublisher.ts

```diff
@@ -117,6 +126,11 @@ EventPublisher.prototype._post = function _post(endpointName, level, group, name
     event.publisher_metadata = this.metadata;
   }
 
+  endpointName === 'EndpointEvents' && this._log.debug(
```

Comment:
Should we just use a traditional `if` statement?

Reply:
Yup, old habits. Actually I already have this fixed locally. Will push up after I fix the tests.